example/lib/widgets/subWidget/event.dart

```diff
@@ -40,7 +40,11 @@ class _EventState extends State<EventHit> {
     var text = "Event sent";
     var subText = "Event has been sent";
     try {
-      await currentVisitor?.sendHit(event);
+      if (currentVisitor != null && currentVisitor.decisionManager.isPanic()) {
```

Comment:
Hum, this means that the QA app will not actually test that the panic mode works, because you do the check of the panic on the QA App side.

Reply:
Maybe we could do the check to display the message, but also send the hit ? so that QA team can check that the event is correctly not sent if panic mode is on

Reply:
Yess , the QA app shouldn't block the button action on sending hits.